Makefile.envs

```diff
@@ -1,5 +1,5 @@
-export EMSCRIPTEN_VERSION = 1.38.44
-export BINARYEN_VERSION = version_89
+export EMSCRIPTEN_VERSION = 2.0.9
+export BINARYEN_VERSION = a8ded16f56afd880a9a6459fe5ce55a8667d9b3e
```

Comment:
emscripten 2.0.12 and binaryen version_99 have recently been released, which covers the changes we need. In particular, it incorporates the packager patch, which you removed